aries-site/src/data/structures/templates.js

```diff
@@ -231,4 +231,24 @@ export const templates = [
     sections: [],
     relatedContent: ['Page Layouts', 'Header', 'Navigation'],
   },
+  {
+    name: 'Table Customization',
```

Comment:
No action item on this yet, just trying to reconcile how this fits into our overall content, its relation to the Table page, inclusion in the Tables page vs. stand-alone ...

If stand-alone, which I think makes sense, are their other table customizations we might add to this page? If so, what are they? If not, should we title this more pointedly (e.g. Ordering Table Columns)? Or is this more about a user's ability to customize their table?

Need to let this sit for a bit. But curious to hear others' thoughts.